### PR TITLE
Make Metadata optional in GTSWrapper

### PR DESCRIPTION
**Pull request by hbs**
This will allow for a more compact serialized form of GTSWrapper instances

### CODE REVIEW

File: warp10/src/main/java/io/warp10/standalone/StandaloneMemoryStore.java [hunk 1]
```diff
@@ -748,10 +749,14 @@ private void load(String path) throws IOException {
         encoder.setCount(wrapper.getCount());
         
         bytes += value.getLength() + key.getLength();
-        encoder.safeSetMetadata(wrapper.getMetadata());
+        if (wrapper.isSetMetadata()) {
+          encoder.safeSetMetadata(wrapper.getMetadata());
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/StandaloneMemoryStore.java]: Same question as previously: is it safe?

File: warp10/src/main/java/io/warp10/standalone/StandaloneChunkedMemoryStore.java [hunk 2]
```diff
@@ -618,10 +619,14 @@ private void load(String path) throws IOException {
         encoder.setCount(wrapper.getCount());
         datapoints += wrapper.getCount();
         bytes += value.getLength() + key.getLength();
-        encoder.safeSetMetadata(wrapper.getMetadata());
+        if (wrapper.isSetMetadata()) {
+          encoder.safeSetMetadata(wrapper.getMetadata());
```
[the remark this answers is not in the document]
Comment (hbs) [reply warp10/src/main/java/io/warp10/standalone/StandaloneChunkedMemoryStore.java]: Yes that's safe as we do not modify further

File: warp10/src/main/java/io/warp10/standalone/StandaloneChunkedMemoryStore.java [hunk 3]
```diff
@@ -532,7 +532,8 @@ public void dump(String path) throws IOException {
 
         for (GTSDecoder decoder: decoders) {
           chunks++;
-          GTSWrapper wrapper = new GTSWrapper(metadata);        
+          GTSWrapper wrapper = new GTSWrapper();
```
[the remark this answers is not in the document]
Comment (hbs) [reply warp10/src/main/java/io/warp10/standalone/StandaloneChunkedMemoryStore.java]: For coherency with other uses

File: warp10/src/main/java/io/warp10/standalone/StandaloneMemoryStore.java (hunk 1 above)
Comment (hbs) [reply warp10/src/main/java/io/warp10/standalone/StandaloneMemoryStore.java]: Same answer ;-)

**Comment by hbs** [comment]
PTAL

**Pull request merged**